editor/js/libs/app.js

```diff
@@ -146,7 +161,23 @@ var APP = {
 
 			for ( var i = 0, l = array.length; i < l; i ++ ) {
 
-				array[ i ]( event );
+				try {
+
+					array[ i ]( event );
+
+				} catch (e) {
+
+					if ( window.console !== undefined && console.error !== undefined ) {
```

Comment:
According to MDN it's not guaranteed to be. As the player seems to be targetting odd devices with touch screens, I figured some odd browser couls be lacking it :-).

    https://developer.mozilla.org/en-US/docs/Web/API/Console/log